src/spawn_players.c

```diff
@@ -22,6 +22,50 @@
 #include "menus.h"
 #include "render_player.h"
 
+f32 D_80165210[8];
+f32 D_80165230[7];
+f32 D_8016524C;
+f32 D_80165250[8];
+s16 D_80165270[8];
+f32 D_80165280[8];
+f32 D_801652A0[8];
+s32 D_801652C0[8];
+s32 D_801652E0[8];
+s16 D_80165300[8];
+// Shadows values from gPathIndexByPlayerId, but is an array
+u16 D_80165310[8];
+// Shadows values from gNearestWaypointByPlayerId, but is an array
+s16 D_80165320[8];
+// Don't think this really belongs with waypoint stuff, but don't know where else to put it
+s16 D_80165330[8];
+s16 D_80165340;
+UNUSED s32 D_80165348[29];
+Player *D_801653C0[8];
+s32 gPlayerIsThrottleActive[8];
+s32 D_80165400[8];
+s32 D_80165420[8];
+s32 D_80165440[8];
+bool gIsPlayerTripleAButtonCombo[8];
+s32 D_80165480[8];
+s32 D_801654A0[8];
+s32 D_801654C0[8];
+s32 D_801654E0[8];
+s32 D_80165500[8];
+s32 D_80165520[8];
+s32 D_80165540[8];
+s16 D_80165560[7];
+s16 D_8016556E;
```

Comment:
It's possible that D_8016556E is apart of the D_80165560 array

Reply:
Took a look at the code. I think your PR is correct

Reply:
I also try but get a missmatch so weird but add a comment to prevent this weird situation and for somone else who have more experience fix it